### DISCUSSION TITLE
Automatic spindown harddrive feature

### DISCUSSION THREAD

**Discussion by ghost** [retronas/discussions#82] [Q&A]
Hi,
I have a gdrive external power hard drive which I use with a raspberry pi. When I'm not using it (during the night), it is still active and spinning full speed. Is it possible to implement a feature to spindown or apply an "idle" state so that it can reduce wear and tear?

**Comment by sairuk** [comment]
its really up to your drive, it should have defaults. 

People have also reported issues with broken samba for console with drives going into standby, we added **optional** config to stop this.

did you install any of the hdparm tool options from the Advanced menu?
if not install hdparm but don't enable any of the option options from the menu and then review the following. Although if so please let us know what.

`/dev/sdX` maybe not be correct for your drive, so substitute as appropriate

`# hdparm -I /dev/sdX` to get information from the drive

You theoretically can manage this manually
`#hdparm -S /dev/sdX` is supposed to put the drive into standby and/or manage the standby lever for the drive

**Comment by ghost** [comment]
Hi,
Thank you for the feedback. From my end, I will put this "issue" on hold, else this "issue" can be put into the Discussion area if anyone else wants to test it
